### DISCUSSION TITLE
Question about synchronization

### DISCUSSION THREAD

**Discussion by dv1** [roc-streaming/roc-toolkit#471] [Q&A]
Do I understand it correctly that ROC does not do _synchronized_ streaming/playback, like AirPlay for example? This would require some form of time synchronization like PTP across senders and receivers to make sure playback starts in sync, and I see nothing like that in ROC. Not saying it is a big problem (I consider it a rather fringe use case), just curious.

**Comment by gavv** [comment]
Hi. Currently roc does not have synchronization feature. I hope it will have it in future, and it's in the roadmap.
